.github/workflows/tilt.yml

```diff
@@ -0,0 +1,40 @@
+name: Tilt CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  tilt:
+    runs-on: tilt-kube-public
+
+    # Cancel previous builds on the same branch/ref. Full runs are expensive
+    # and capacity is limited, so we want to avoid running multiple builds
+    # in parallel even if it means skipping CI runs on permanent branches
+    # (unfortunately, we can't differentiate between temporary and permanent
+    # refs without duplicating the entire logic).
+    concurrency:
+      group: ${{ github.workflow }}-tilt-${{ github.ref }}
+      cancel-in-progress: true
+
+    steps:
+      - name: Clear repository
+        run: |
+          rm -rf $GITHUB_WORKSPACE && mkdir $GITHUB_WORKSPACE
+      - uses: actions/checkout@v2
+      - name: Expand for link to Tilt dashboard (only available during build)
+        run: >
+          echo "Tilt progress dashboard: https://$DASHBOARD_URL"
+      - run: |
+          kubectl config set-context ci --namespace=$DEPLOY_NS
+          kubectl config use-context ci
+
+      - run: tilt ci -- --evm2 --namespace=$DEPLOY_NS
+        timeout-minutes: 60
```

Comment:
The default timeout for `tilt ci` is 30 mins btw despite having the longer timeout on the step. Don't think we should need to increase beyond 30 though

Reply:
agree, this was copied from the wormhole repo. would rather have `tilt ci` timeout than GH kill the step, I guess :shrug: open to changing in a future PR